phylogenetic/rules/prepare_sequences.smk

```diff
@@ -1,22 +1,86 @@
 """
 This part of the workflow prepares sequences for constructing the phylogenetic tree.
 
-REQUIRED INPUTS:
-
-    metadata    = data/metadata.tsv
-    sequences   = data/sequences.fasta
-    reference   = ../shared/reference.fasta
-
-OUTPUTS:
-
-    prepared_sequences = results/prepared_sequences.fasta
+See Augur's usage docs for these commands for more details.
+"""
+rule download:
+    """Downloading sequences and metadata from data.nextstrain.org"""
+    output:
+        sequences = "data/sequences.fasta.zst",
+        metadata = "data/metadata.tsv.zst"
+    params:
+        sequences_url = "https://data.nextstrain.org/files/workflows/rabies/sequences.fasta.zst",
+        metadata_url = "https://data.nextstrain.org/files/workflows/rabies/metadata.tsv.zst"
+    shell:
+        """
+        curl -fsSL --compressed {params.sequences_url:q} --output {output.sequences}
+        curl -fsSL --compressed {params.metadata_url:q} --output {output.metadata}
+        """
 
-This part of the workflow usually includes the following steps:
+rule decompress:
+    """Decompressing sequences and metadata"""
+    input:
+        sequences = "data/sequences.fasta.zst",
+        metadata = "data/metadata.tsv.zst"
+    output:
+        sequences = "data/sequences.fasta",
+        metadata = "data/metadata.tsv"
+    shell:
+        """
+        zstd -d -c {input.sequences} > {output.sequences}
+        zstd -d -c {input.metadata} > {output.metadata}
+        """
 
-    - augur index
-    - augur filter
-    - augur align
-    - augur mask
+rule filter:
+    """
+    Filtering to
+      - {params.sequences_per_group} sequence(s) per {params.group_by!s}
+      - from {params.min_date} onwards
+      - excluding strains in {input.exclude}
+      - minimum genome length of {params.min_length}
+    """
+    input:
+        sequences = "data/sequences.fasta",
+        metadata = "data/metadata.tsv",
```

Comment:
should these be config values, to allow somebody to override them?

Reply:
Good question. I think this comes back to the bigger question of how much should be in the config file.

The norm in the phylogenetic workflows has been to start with `data/sequences.fasta` + `data/metadata.tsv`, so it's the [suggested framework in the pathogen-repo-guide](https://github.com/nextstrain/pathogen-repo-guide/blob/89b3c5dbc9b8f6a009f4a19c3ac56113bc5511ee/phylogenetic/rules/prepare_sequences.smk#L4-L8).

Reply:
I decided to stick with the norms.